### PR TITLE
Second Question added 

### PR DESCRIPTION
**Pull request by vb-vinay**
One question is added and spelling of 'portrays' is corrected

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,13 +1,14 @@
...
 
-At the time of joining, the HR potrays the company like it is match made in Heaven, but sometimes reality is far from it. I have had my experiences, atleast you should be aware of what you are getting into.
+At the time of joining, the HR portrays the company like it is match made in Heaven, but sometimes reality is far from it. I have had my experiences, atleast you should be aware of what you are getting into.
 
 ## :question: Questions
 
 | S.No. |Question |
 |---|---|
 | 1 | Will the Notice period mentioned in the Offer letter, can be increased in future(Actually happened to me)? |
+| 2 | Is it valid to change the probation period from 6 months (as mentioned by the HR on the Interview Day) to "12 months", in the offer letter which is received 1 month after the Interview and is just a day less than the joining date?
```
Comment (sanjeevyadavIT) [inline README.md]: Please change the correct question number

**Review by sanjeevyadavIT — CHANGES_REQUESTED** [review]
Update the question number, and resolve the conflict

**Comment by vb-vinay** [comment]
The question numbers have been changed and conflict is resolved

**Review by rishabh115 — APPROVED** [review]
Fixed the question number issue.

**Pull request merged**